### PR TITLE
Prep for PyPi Upload

### PR DESCRIPTION
**Pull request by tyhoff**
Clean up and add a few things to the `setup.py` that are generally advised for uploading packages to PyPi. Honestly not much here otherwise. 

Set the initial version to 0.1.0 as a good starting point

> Depends on #35. Will rebase against that change when it's merged.

### CODE REVIEW

**Review by dkalowsk — APPROVED** [review]
Not sure if this will work, but it seems this PR is in need of an approval.  Let's see if that will move it forward or has puncover become completely abandoned.

**Comment by noahp** [comment]
Merging as is! gonna touch up after merge and ship a new python3.9 + python3.10 release!

**Comment by noahp** [comment]
Merged these changes as #53

**Pull request closed without merging**
